recipes/fontconfig/all/conanfile.py

```diff
@@ -0,0 +1,84 @@
+    description = "Fontconfig is a library for configuring and customizing font access"
+    homepage = "https://gitlab.freedesktop.org/fontconfig/fontconfig"
+    topics = ("conan", "fontconfig", "fonts", "freedesktop")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    generators = "pkg_config"
+    _source_subfolder = "source_subfolder"
+    _autotools = None
+
+    def requirements(self):
+        self.requires("freetype/2.10.1")
+        self.requires("expat/2.2.9")
+        if self.settings.os == "Linux":
+            self.requires("libuuid/1.0.3")
+
+    def configure(self):
+        if self.settings.os == "Windows":
+            raise ConanInvalidConfiguration("Windows builds are not supported.")
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extrated_dir = self.name + "-" + self.version
+        os.rename(extrated_dir, self._source_subfolder)
+
+    def build_requirements(self):
+        self.build_requires("gperf/3.1")
+
+    def _configure_autotools(self):
+        if not self._autotools:
+            args = ["--enable-static=%s" % ("no" if self.options.shared else "yes"),
+                    "--enable-shared=%s" % ("yes" if self.options.shared else "no"),
+                    "--disable-docs"]
+            self._autotools = AutoToolsBuildEnvironment(self)
+            self._autotools.configure(configure_dir=self._source_subfolder, args=args)
+            tools.replace_in_file("Makefile", "po-conf test", "po-conf")
+        return self._autotools
+
+    def _patch_files(self):
+        def rename_if_exists(old, new):
+            if os.path.exists(old):
+                os.rename(old, new)
+        #  - fontconfig requires libtool version number, change it for the corresponding freetype one
+        tools.replace_in_file(os.path.join(self._source_subfolder, 'configure'), '21.0.15', '2.8.1')
+        rename_if_exists('freetype.pc', 'freetype2.pc')
+
+    def build(self):
+        # Patch files from dependencies
+        self._patch_files()
+        autotools = self._configure_autotools()
+        autotools.make()
+
+    def package(self):
+        self.copy("COPYING", dst="licenses", src=self._source_subfolder)
+        autotools = self._configure_autotools()
+        autotools.install()
+        la = os.path.join(self.package_folder, "lib", "libfontconfig.la")
+        if os.path.isfile(la):
+            os.unlink(la)
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        tools.rmdir(os.path.join(self.package_folder, "etc"))
+        tools.rmdir(os.path.join(self.package_folder, "share"))
+        tools.rmdir(os.path.join(self.package_folder, "var"))
+
+    def package_info(self):
+        self.cpp_info.libs = tools.collect_libs(self)
```

Comment:
The output of a random run showed me that only one library was packaged.
```suggestion
        self.cpp_info.libs = ["fontconfig"]
```